x-pack/plugins/canvas/tsconfig.json

```diff
@@ -8,6 +8,7 @@
     "declarationMap": true
   },
   "include": [
+    "../../../node_modules/@types/react-syntax-highlighter/**/*",
```

Comment:
Without this addition, the `scripts/type_check` check would give the following error:

```
ERROR x-pack/plugins/canvas failed
      ../../../kibana/node_modules/@storybook/components/dist/syntaxhighlighter/syntaxhighlighter.d.ts:2:36 - error TS7016: Could not find a declaration file for module 'react-syntax-highlighter/dist/cjs/prism-light'. '/dev/shm/workspace/kibana/node_modules/react-syntax-highlighter/dist/cjs/prism-light.js' implicitly has an 'any' type.
        Try `npm i --save-dev @types/react-syntax-highlighter` if it exists or add a new declaration (.d.ts) file containing `declare module 'react-syntax-highlighter/dist/cjs/prism-light';`

      2 import ReactSyntaxHighlighter from 'react-syntax-highlighter/dist/cjs/prism-light';
                                           ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~


      Found 1 error.
```

I'm not comfortable that this fix is the right one, but it does work.

Reply:
@elastic/kibana-core what do you think? ☝️

Reply:
@elastic/kibana-presentation team has already experienced a similar problem https://github.com/elastic/kibana/blob/dd118e2c86f2f628d39d197c7d188ccaef7dc43b/typings/index.d.ts#L27-L32 
It seems to be a problem in Storybook generated `d.ts` files, since it adds `@ts-ignore` to the module import https://github.com/storybookjs/storybook/blob/dd7da2000ff45aaba37232260a7ee7e473bc416d/lib/components/src/syntaxhighlighter/syntaxhighlighter.tsx#L30-L31

Reply:
Nicely spotted! I reverted my previous fix and pushed a new commit with a line added to this file 👍